static/benchmarks/css/table.sass

```diff
@@ -216,3 +216,9 @@ input[type=checkbox]
 // children count
 .benchmark-count
   margin-left: 5px
+
+span.bold
+  font-weight: 700
+
+span.italic
+  font-style: italic
```

Comment:
I prefer _semantic_ names for the CSS classes so that HTML doesn't define the style (https://dev.to/sargalias/why-you-should-use-semantic-naming-in-css-1dik).
Could you rename these to what they correspond to, e.g. `span.information` or so?